wordpress-mobile/gutenberg-mobile
# Correct buttons appender

Fixes #2227 
Ref to gutenberg: https://github.com/WordPress/gutenberg/pull/22362
To test:

1.Add Buttons block
2.Type Hello Hello Hello in first Button
3.Press (+) appender to add next Button
4.Expect that appender drop to the next line
5.Press (+) appender
6.Expect appender is clickable and another Button was added

PR submission checklist:
